.github/workflows/build_dev_documentation.yml

```diff
@@ -110,14 +110,14 @@ jobs:
 
       - name: Make documentation
         run: |
-          cd doc-builder
-          doc-builder build transformers ../transformers/docs/source --build_dir ../doc-build-dev --notebook_dir notebooks/transformers_doc --clean --version pr_$PR_NUMBER --html
+          cd doc-builder &&
```

Comment:
The `&&` ensure that an error in the commands will mark the step as failed and stop the CI